corgi/core/models.py

```diff
@@ -1696,13 +1696,17 @@ def get_provides_nodes_queryset(
         )
         if include_dev:
             type_list = ComponentNode.PROVIDES_NODE_TYPES
-        return (
-            self.cnodes.get_queryset()
-            .get_descendants()
-            .filter(type__in=type_list)
-            .prefetch_related("obj")
-            .using(using)
-        )
+
+        provides_set = set()
+        for cnode in self.cnodes.get_queryset():
+            provides_set.update(
+                cnode.get_descendants()
+                .filter(type__in=type_list)
+                .prefetch_related("obj")
```

Comment:
We're not accessing the obj here anymore, it's only used in the queryset below:
```suggestion
                cnode.get_descendants()
                .filter(type__in=type_list)
```